sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -246,12 +264,11 @@ static const struct snd_soc_dapm_route map[] = {
 	/* Headphones */
 	{ "Headphone", NULL, "rt711 HP" },
 	{ "rt711 MIC2", NULL, "Headset Mic" },
-	/* Speakers */
-	{ "Speaker", NULL, "rt1308-1 SPOL" },
-	{ "Speaker", NULL, "rt1308-1 SPOR" },
 };
 
-static const struct snd_soc_dapm_route second_speaker_map[] = {
+static const struct snd_soc_dapm_route rt1308_speaker_map[] = {
+	{ "Speaker", NULL, "rt1308-1 SPOL" },
+	{ "Speaker", NULL, "rt1308-1 SPOR" },
```

Comment:
please add a big fat comment that the order of the definitions matter, and that the last two routes will not be added in the single amplifier case.